medicinskij-academicheskij-jurnal.csl

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Медицинский академический журнал (Russian)</title>
```

Comment:
@ValeryMukhin, in the CSL styles repository, we always make sure that the language in the style's title ("(Russian)" here) matches the value of the "default-locale" attribute.

If you wish to keep the style set to US English, please remove the " (Russian)".